freqtrade/exchange/exchange.py

```diff
@@ -165,6 +165,11 @@ def id(self) -> str:
         """exchange ccxt id"""
         return self._api.id
 
+    @property
+    def markets(self) -> Dict:
+        """exchange ccxt markets"""
+        return self._api.markets
```

Comment:
I think we should have  a check here if markets was initialized (maybe load_markets did fail for whatever reason).
in that rare case we should probably log and "retry" to call load_markets...